glean-core/ios/Glean/Glean.swift

```diff
@@ -84,10 +86,18 @@ public class Glean {
             self.pingTypeQueue.removeAll()
         }
 
+        initializeCoreMetrics()
+
+        // Deal with any pending events so we can start recording new ones
+        glean_on_ready_to_send_pings(self.handle)
+
         // Signal Dispatcher that init is complete
         Dispatchers.shared.flushQueuedInitialTasks()
+    }
 
-        self.initialized = true
+    /// Initialize the core metrics internally managed by Glean (e.g. client id).
+    private func initializeCoreMetrics() {
+        /// left empty for now, we don't have additional core metrics
```

Comment:
What's needed here? Is there a bug for implementing these? Can you reference it here?